openbb_terminal/cryptocurrency/defi/defipulse_model.py

```diff
@@ -43,11 +43,31 @@ def get_defipulse_index(sortby: str = "TVL", ascend: bool = False) -> pd.DataFra
             "Name",
             "Chain",
             "Sector",
-            "TVL",
-            "1 Day (%)",
+            "30D_Users",
+            "TVL_$",
+            "1_Day_%",
         ],
     )
+
     df["Rank"] = df.index
-    df = df.sort_values(by=sortby, ascending=ascend)
+    df["30D_Users"] = df["30D_Users"].str.replace(",", "").astype(int)
+    for i, row in df.iterrows():
+        value = row["TVL_$"]
+        residue = len(value.partition(".")[2]) - 1
+        value = value.replace("$", "")
+        value = value.replace("M", "000000")
+        value = value.replace("B", "000000000")
+        value = value[0:-residue]
+        value = value.replace(".", "")
+        df.at[i, "TVL_$"] = int(value)
 
+        percent = row["1_Day_%"]
+        percent = percent.replace("%", "")
+        percent = percent.replace("+", "")
```

Comment:
For future reference,. there is already a helper_func: `lambda_clean_data_values_to_float` that you can just use to turn the string into a number